src/cli.js

```diff
@@ -21,7 +21,8 @@ const printHelp = function() {
   console.log('  -V, --verbose       Print all results, even successful ones.');
   console.log('  -q, --quiet         Don\'t print anything.');
   console.log('  -d, --dir [dir]     Run in this project directory, defaults to cwd.');
-  console.log('  --skip-es-check     Do not run es check on dist, for projects with no dist.');
+  console.log('  --skip-es-check     skip the syntax check, still here for backwards compatablity');
+  console.log(`  --skip-[name]       skip a test that you do not want to run. ${testNames.join(', ')}`);
```

Comment:
This option replaces `--skip-es-check` and supports any of the tests that we run. I am keeping `--skip-es-check` around to keep this fairly backwards compatible.